example_site/src/components/AccountPage.js

```diff
@@ -65,7 +62,8 @@ export default function AccountPage() {
 
       <button 
         className="submit-button configuration-submit-button"
-        onClick={onResetWaymarkInstance}>
+        onClick={onResetWaymarkInstance}
```

Comment:
we need to `await` these calls! I think what's happening with the 'back to start' flow is that we are not properly waiting for the old waymark instance to be cleaned up before creating a new one and then the frame communicator is not being instantiated correctly.